emfacilities/protocols/protocol_data_counter.py

```diff
@@ -0,0 +1,302 @@
+# **************************************************************************
+# *
+# * Authors: Daniel Marchan (da.marchan@cnb.csic.es)
+# *
+# * Unidad de  Bioinformatica of Centro Nacional de Biotecnologia , CSIC
+# *
+# * This program is free software; you can redistribute it and/or modify
+# * it under the terms of the GNU General Public License as published by
+# * the Free Software Foundation; either version 2 of the License, or
+# * (at your option) any later version.
+# *
+# * This program is distributed in the hope that it will be useful,
+# * but WITHOUT ANY WARRANTY; without even the implied warranty of
+# * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# * GNU General Public License for more details.
+# *
+# * You should have received a copy of the GNU General Public License
+# * along with this program; if not, write to the Free Software
+# * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+# * 02111-1307  USA
+# *
+# *  All comments concerning this program package may be sent to the
+# *  e-mail address 'scipion@cnb.csic.es'
+# *
+# **************************************************************************
+import os
+from datetime import datetime, timedelta
+import time
+import copy
+import re
+
+from pwem.protocols import EMProtocol
+from pwem.objects import SetOfImages, Set
+
+from pyworkflow import VERSION_3_0
+import pyworkflow.protocol.params as params
+import pyworkflow.utils as pwutils
+from pyworkflow import UPDATED, NEW
+from pyworkflow.protocol.constants import STATUS_NEW
+
+
+
+OUTPUT = "outputSet"
+
+class ProtDataCounter(EMProtocol):
+    """
+    Protocol to make a subset of images from the original one.
+    Waits until certain number of images is prepared and then send them to output.
+    It can be done in 3 ways:
```

Comment:
Which three ways?. Dont add artificial lines break because make less readable the help